client/src/Components/Employees/EmployeeSingle.js

```diff
@@ -1,19 +1,158 @@
 import React, { Component } from "react";
 // import { Route, Link, withRouter } from "react-router-dom";
-import { Button, Icon } from "semantic-ui-react";
-import { FlexRow, Container } from "custom-components";
-import Card from "../shared/Card/Card";
+import { Button, Icon, Image, Header, Dropdown } from "semantic-ui-react";
+import { FlexRow, Container, FlexColumn } from "custom-components";
+
+import taskPropertyAssign from "./taskPropertyHelper";
 
 class EmployeeSingle extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.loading = true;
+  }
+
+  componentDidMount() {
+    if (!this.props.employees.length) {
+      const query = {
+        page: 1,
+        pageSize: 10000,
+        sort: "_id",
+        search: ""
+      };
+      this.props.getEmployees({ ...query });
+    }
   }
 
-  componentDidMount() {}
+  permissionChange = body => {
+    this.props.updateEmployee(this.props.match.params.id, body);
+  };
 
   render() {
-    return <Container>{/* <div>this.state.name</div> */}</Container>;
+    const employees = taskPropertyAssign({
+      employees: this.props.employees,
+      tasks: this.props.tasks,
+      properties: this.props.properties
+    });
+    let permissions;
+    const id = this.props.match.params.id;
+    const employee = employees.find(item => item._id === id);
+    if (employee) {
+      this.loading = false;
+      permissions = employee.permissions;
+    }
+    const permissionValues = [
+      {
+        key: "yes",
+        text: "Yes",
+        value: true
+      },
+      {
+        key: "no",
+        text: "No",
+        value: false
+      }
+    ];
+
+    return (
+      <Container>
+        {this.loading ? (
+          <div>Please wait...</div>
+        ) : (
+          <FlexColumn justifyCenter alignStart width="full">
+            <FlexRow justifyAround alignCenter width="full">
+              <FlexColumn alignStart>
+                <Header as="h1">
+                  {" "}
+                  {employee.firstName} {employee.lastName}{" "}
+                </Header>
+                <br />
+                <br />
+                <br />
+                <Header as="h3"> Email: {employee.email} </Header>
+                <Header as="h3"> Phone: {employee.phone} </Header>
+                <br />
+                <Header as="h3">Today's Tasks: {employee.todayTask}</Header>
```

Comment:
Adjust this header's text content to be inside of back ticks to prevent an issue with open quotation

Reply:
Done